### PR TITLE
feat: add max/min option for number-based options

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

**Reference Discord API Docs PRs or commits:**
- https://github.com/discord/discord-api-docs/pull/3967

### CODE REVIEW

**Review by vladfrangu — CHANGES_REQUESTED** [review]
Before I can approve this, can you make sure this works with autocomplete types?

**Comment by suneettipirneni** [comment]
> Before I can approve this, can you make sure this works with autocomplete types?

Quick question, can max/min work with autocompleted values?

**Comment by vladfrangu** [comment]
I don't know, try it 😅

**Comment by suneettipirneni** [comment]
autocomplete seems to work with min/max, so I adjusted the types for this

File: deno/payloads/v8/_interactions/_applicationCommands/chatInput.ts [hunk 1]
```diff
@@ -89,13 +92,24 @@ export interface APIApplicationCommandArgumentOptions
...
+ *
+ * In contrast to `APIApplicationCommandArgumentOptions`, these types cannot have an `choices` array,
+ * but they can a `autocomplete` field where it's set to `true`
+ */
+export interface APIApplicationCommandNumericAutocompleteOptions
+	extends Omit<APIApplicationCommandOptionBase, 'type' | 'autocomplete'> {
+	type: ApplicationCommandOptionType.Integer | ApplicationCommandOptionType.Number;
 	autocomplete: true;
+	max_value?: number;
+	min_value?: number;
```
Comment (vladfrangu) [inline deno/payloads/v8/_interactions/_applicationCommands/chatInput.ts:112]: Do documentations carry over?

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**